.github/workflows/release-ticket-creation.yml

```diff
@@ -3,33 +3,36 @@ on:
   workflow_dispatch:
     inputs:
       branch_name:
-        description: 'Release target branch name'
+        description: 'Type a branch name starting with `release/v`'
         required: true
 
 jobs:
   trigger-release-ticket-creation:
     name: Trigger release ticket creation
     runs-on: ubuntu-latest
-    # The branch name should starts with 'release/v'
-    if: startsWith(github.event.inputs.branch_name, 'release/v')
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+
+      - name: Verify branch name
```

Comment:
Why I changed the one line cmd like this? 
With 
```
if: startsWith(github.event.inputs.branch_name, 'release/v')
```
the workflow will skipped in silence but I want to make it failed \w the error message so we know why it failed.